modules/Microsoft.Network/privateEndpoints/privateDnsZoneGroups/deploy.bicep

```diff
@@ -1,10 +1,12 @@
 @description('Conditional. The name of the parent private endpoint. Required if the template is used in a standalone deployment.')
 param privateEndpointName string
 
-@description('Required. List of private DNS resource IDs.')
+@description('Required. Array of private DNS zone resource IDs. Each DNS zone group can support up to 5 DNS zones.')
+@minLength(1)
+@maxLength(5)
 param privateDNSResourceIds array
 
-@description('Optional. The name of the private DNS Zone Group.')
+@description('Optional. The name of the private DNS zone group.')
```

Comment:
Why do you change the casing? Did we as some point agree on how we treat Azure resource names (e.g., always lower-case, or upper-case)? Maybe I forgot.

Reply:
We didn't. Just any other occurrence of `zone group` is lower-case throughout all files.

Reply:
Alright something for the future then 😊